Makefile

```diff
@@ -1,5 +1,5 @@
 GO=$(shell which go)
-OTELCOL_BUILDER_VERSION ?= 0.81.0
+OTELCOL_BUILDER_VERSION ?= 0.86.0
```

Comment:
[0.87.0](https://github.com/open-telemetry/opentelemetry-operator/releases/tag/v0.87.0). Should we bump to it?

Reply:
I think the important part is that its >= 0.86.0 that to replace logging with debug. We will bump it before the 3.0 release anyway.